### ISSUE TITLE
docs: consistently use "suite" keyword in documentation.

### ISSUE THREAD

**Issue by ealush** [ealush/vest#602]
The documentation has different terms describing Vest's suite. Sometimes we reference it as "validate", sometimes as "v". The idea is to name the suite's variable and module as "suite" consistently across the docs.

## From:
```js
const validate = vest.create(() => {
  // ...
});
// or 
const v = vest.create(() => {
  // ...
});
```

## To:
```js
const suite = vest.create(() => {
  // ...
});
```

Here are some docs that I've found:
https://github.com/ealush/vest/blob/latest/packages/vest/docs/cross_field_validations.md
https://github.com/ealush/vest/blob/latest/packages/vest/docs/exclusion.md
https://github.com/ealush/vest/blob/latest/packages/vest/docs/getting_started.md
https://github.com/ealush/vest/blob/latest/packages/vest/docs/group.md
https://github.com/ealush/vest/blob/latest/packages/vest/docs/result.md
https://github.com/ealush/vest/blob/latest/packages/vest/docs/utilities.md
https://github.com/ealush/vest/blob/latest/packages/vest/docs/warn.md

**Comment by galitpauz** [comment]
@ealush Is this taken?

**Comment by ealush** [comment]
@galitpauz, no, this one's not taken yet. You can go ahead and do it. Should I assign this one to you?

**Comment by galitpauz** [comment]
@ealush Yes, thank you.

**Comment by ealush** [comment]
Done

**Comment by ealush** [comment]
Closed by #610, thanks @galitpauz!
